docs/antora/ccutil/guides-server/master.adoc

```diff
@@ -0,0 +1,7 @@
+:toc:
+:toclevels: 3
+:numbered:
+:linkattrs:
+
+include::guides-server/20.0/guides-keycloak-server-ccutil.adoc[]
```

Comment:
As above `{project_versionDoc}`?

Reply:
done, this is now using an unversioned Antora component, so it is no longer part of the path.